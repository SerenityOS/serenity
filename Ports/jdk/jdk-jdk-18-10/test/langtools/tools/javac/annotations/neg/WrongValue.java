/*
 * @test /nodynamiccopyright/
 * @bug 4951298
 * @summary compiler: crashes when attributes with same elements are used in place of other
 * @author gafter
 *
 * @compile/fail/ref=WrongValue.out -XDrawDiagnostics  WrongValue.java
 */

@interface TestM2 {
    String message() default "MyMessage";
    Class myClass() default Integer.class;
}

@interface TestM3 {
    String message() default "MyMessage";
    Class myClass() default Integer.class;

}

@interface X {
    TestM2 value();
}

@X(@TestM3())
class Foo {
}
