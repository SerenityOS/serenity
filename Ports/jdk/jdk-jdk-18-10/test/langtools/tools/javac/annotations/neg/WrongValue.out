WrongValue.java:25:5: compiler.err.prob.found.req: (compiler.misc.inconvertible.types: TestM3, TestM2)
1 error
